CHANGELOG.md

```diff
@@ -4,13 +4,21 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.5.2] (Unreleased)
+## [0.6.0] (Unreleased)
```

Comment:
0.5.2 was going to be a patch release because since it just fixed a bug. But now I'm adding an external method so that felt like a minor version bump was appropriate. (Note that 0.5.2 was never released.)

Reply:
It's cool that I wrote the CHANGELOG copy first. That's what prompted me to change the version number. Then I went and bumped it.